### ISSUE TITLE
Multiple endpoint

### ISSUE THREAD

**Issue by teopost** [Netflix-Skunkworks/go-jira#138]
Hi,
  It is possible configure multiple endpoint?

**Comment by coryb** [comment]
You can only configure one endpoint per operation, but you can configure different endpoints per operation.   Typically you just want to have different endpoints for different projects, in which case you just need to create a `.jira.d/config.yml` at the root of your project directory with an `endpoint: https://jira` configuration.

If for some reason you want to have a `jira foo` command go against a different endpoint from a `jira bar` command you can create `.jira.d/foo.yml` and `.jira.d/bar.yml` both with different `endpoint` configs.  Of you can just override it on the command line with `-e`.

**Comment by teopost** [comment]
I work for 3 company. I have 3 account and 3 endpoint for 3 jira
installation.
I can connect to 3 different jira installation ?

Il 06 gen 2018 23:34, "coryb" <notifications@github.com> ha scritto:

**Comment by coryb** [comment]
Yes, you just need 3 different `.jira.d/config.yml` files.  `go-jira` allows for configuration in multiple locations.  So for your 3 companies, I assume you have some directory structure on your computer something like:
```
$HOME/company1/project1
$HOME/company1/project2

$HOME/company2/project1
$HOME/company2/project1

$HOME/company3/project1
$HOME/company3/project2
```

In this scenario you would create `$HOME/company1/.jira.d/config.yml` to have the `endpoint` and `user` properties for company1, then similarly create `$HOME/company2/.jira.d/config.yml` and `$HOME/company3/.jira.d/config.yml`

Then when you run `jira` anywhere under the `$HOME/company1` directory the config in `$HOME/company1/.jira.d/config.yml` will get processed.

Basically `go-jira` will look through all parent directories looking for all the `.jira.d/config.yml` files and merge them where the closest file take precedence for configuration.

**Comment by teopost** [comment]
Wow! Is fantastic.
It works!
Thank you
